fabric-serialization-api-v1/build.gradle

```diff
@@ -0,0 +1,11 @@
+version = getSubprojectVersion(project)
+
+loom {}
+
+moduleDependencies(project, [
+	':fabric-api-base',
+])
```

Comment:
Is this used?

Reply:
To provide a codec for TriState, but honestly that might be moved directly to base module

Reply:
Ah thats fine, I tottaly missed that. Id leave it as-is.

Reply:
This would make sense, but adds extra dependencies to it (through would be useful either way)